lib/paychex/client/companies.rb

```diff
@@ -85,5 +115,13 @@ def details_by_display_ids(display_ids)
         ret
       end
     end
+
+    def companies(options)

```

Comment:
In this case, I think we could mark `companies` as private?

Reply:
Done. Thank you.